_posts/website/_posts/2018-10-31-tiaas-feedback.md

```diff
@@ -0,0 +1,17 @@
+---
+site: freiburg
+title: 'User feedback from Maria Doyle'
+external: "https://www.petermac.org/"
```

Comment:
I'm not sure we should use external here, that makes this  not a post but a redirect to their site. I don't think that makes sense in this case.